src/label/creator.py

```diff
@@ -397,3 +397,11 @@ def create_name_tag(member_id, name, membership_end_date):
               LabelString(f'{name}'),
               LabelString(membership_string)]
     return Label(labels)
+
+
+def create_meetup_name_tag(name):
+
+    labels = [LabelString(f'{name}'),
+              LabelString('Ask me about this:'),
```

Comment:
Jag tycker det låter bättre med: 
```suggestion
              LabelString('Ask me about:'),
```